src/components/Avatar/Avatar.tsx

```diff
@@ -124,7 +124,7 @@ const AvatarBase = styled.button<IStyled<AvatarBaseProps>>(
   height: ${props.customProps.size};
 
   overflow: hidden;
-  border-radius: 50%;
+  border-radius: 99999px;
```

Comment:
Looks a bit hacky

Reply:
Yes.. on more digging find out.. we don't have to change this property. `overflow: hidden` property of button component is making it shrink when it's container is flex.